src/main/java/com/faforever/client/login/LoginController.java

```diff
@@ -440,7 +441,16 @@ private void showLoginProgess() {
   }
 
   public void onPlayOfflineButtonClicked() {
-    gameService.startGameOffline();
+    try {
+      gameService.startGameOffline();
+    } catch (Exception e) {
+      if (e.getCause() instanceof IOException) {
+        notificationService.addImmediateWarnNotification("offline.noExe");
```

Comment:
If I remove the exe I don't get the error message, but instead I get prompted to locate the forged alliance directory.

Reply:
Can you verify that the game path didn't get removed from your prefs by some test. You will get the game directory pop up if it doesn't know where the supcom install directory is. This logic happens in the game service.

Reply:
Seems like it happened because I just have the exe on linux to trick the client. I just tested it on windows with a regular faf install and there it works as expected when I rename the exe.